### ISSUE TITLE
droidbot_ime is failing to write text which contains space

### ISSUE THREAD

**Issue by dridiha** [honeynet/droidbot#143]
Hello,

I want to report an issue within droidbot, exactlyin droidbot_ime, It appears to be unable to write a text that contains a space character.
I hope you look into this issue.

Thanks

**Comment by dridiha** [comment]
Hi,
To correct this, you have just to change the line L89 with:
input_cmd = ["am", "broadcast", "-a", "DROIDBOT_INPUT_TEXT", "--es", "text", text, "--ei", "mode", str(mode)]

**Comment by freddiewanah** [comment]
Hi, I think the problem has appeared again.

When I tried to run the command say:
adb shell am broadcast -a DROIDBOT_INPUT_TEXT --es text 'football club' --ei mode 0 

It responds the following:
Broadcasting: Intent { act=DROIDBOT_INPUT_TEXT flg=0x400000 pkg=club (has extras) }
Broadcast completed: result=0

**Comment by yuanchun-li** [comment]
Yeah. I also noticed this issue. Unfortunately I don't have a solution yet. A dirty workaround is to avoid using spaces in the input text. For example, use "football-club" instead of "football club".
It could be fixed by adding some logic to replace "-" with spaces in DroidBotIME. I will do it when I have time.

**Comment by yuanchun-li** [comment]
It should be fixed in the latest commit c958a0b
